Closes #1859 metadata updated for DAIDs in xlsx file (#2060)
pharmaverse/admiral: NEWS.md

than one observation, e.g., events which need to be confirmed by a second
assessment. (#1960)

> - `data/atoxgr_criteria_daids.rda` added, which holds metadata for [Division of AIDS (DAIDS) Table for Grading the Severity of Adult and Pediatric Adverse Events](https://rsc.niaid.nih.gov/sites/default/files/daidsgradingcorrectedv21.pdf)

Reviewer: ```suggestion
- `atoxgr_criteria_daids.rda` added, which holds metadata for [Division of AIDS (DAIDS) Table for Grading the Severity of Adult and Pediatric Adverse Events](https://rsc.niaid.nih.gov/sites/default/files/daidsgradingcorrectedv21.pdf). You can find additional documentation here `atoxgr_criteria_daids()`
```

> 

Reviewer: So the issue for this is pretty bland...but I think the discussion in the PR is very interesting.  Shall we link the PR number here?

Author: we should add the link to the PR under the new argument `signif_dig` for `derive_var_atoxgr_dir`, right? As the significant digits was the main discussion point in thre PR?

    
    - The `description` field was added to `event()`. It can be used to provide
    a description of the event in plain language.
    
- `derive_var_atoxgr_dir()` was enhanced (#1859):
  

>   - can now select `atoxgr_criteria_daids` in argument `meta_criteria` to create `ATOXGRL` and `ATOXGRH` based on [Division of AIDS (DAIDS) Table for Grading the Severity of Adult and Pediatric Adverse Events](https://rsc.niaid.nih.gov/sites/default/files/daidsgradingcorrectedv21.pdf)

Reviewer: ```suggestion
  -  Can now select `atoxgr_criteria_daids` in argument `meta_criteria` to create `ATOXGRL` and `ATOXGRH` based on [Division of AIDS (DAIDS) Table for Grading the Severity of Adult and Pediatric Adverse Events](https://rsc.niaid.nih.gov/sites/default/files/daidsgradingcorrectedv21.pdf)
```

Reviewer: I think if we add `()` then it will auto-link to the reference page

Author: Don't think we should add the '()' as thats  not allowed in the argument, just the text `atoxgr_criteria_daids`. I capitalised the first letter at start of sentence, ie `Can`

>   - new argument `signif_dig` added to control the number of significant digits to use when comparing 2 numeric values.

Reviewer: ```suggestion
  - New argument `signif_dig` added to control the number of significant digits to use when comparing 2 numeric values.
```
